pkg/doc/verifiable/credential.go

```diff
@@ -0,0 +1,271 @@
+/*
```

Comment:
suggest squashing the commits in the PR.

Reply:
Done